lib/api/CDataFrameOutliersRunner.cc

```diff
@@ -123,10 +99,13 @@ void CDataFrameOutliersRunner::writeOneRow(const TStrVec& featureNames,
 }
 
 void CDataFrameOutliersRunner::runImpl(core::CDataFrame& frame) {
-    maths::COutliers::SComputeParameters params{
-        this->spec().numberThreads(), this->numberPartitions(),
-        m_ComputeFeatureInfluence, m_ProbabilityOutlier};
-
+    maths::COutliers::SComputeParameters params{this->spec().numberThreads(),
+                                                this->numberPartitions(),
+                                                m_StandardizeColumns,
+                                                static_cast<maths::COutliers::EMethod>(m_Method),
+                                                m_NumberNeighbours,
+                                                m_ComputeFeatureInfluence,
+                                                m_ProbabilityOutlier};
```

Comment:
`m_ProbabilityOutlier` it seems to me that this would be good to have as a parameter as well. Do we actually use this in our `lof` calculation?

Reply:
I agree. We use it in the ensemble as the prior probability something is an outlier.

(It doesn't really enter directly into LOF as a parameter. The sklearn contamination parameter to LOF acts (somewhat equivalently) on a decision_function and not the raw calculation of outlier factors. There are potentially other ways we could use it, but some of these would assume that it is reasonably accurate and it is unlikely that this quantity is accurately known in advance. This might be something we revisit though.)